scripts/get_manifest.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+# This script has the following arguments:
+#     - release tag,
+# ./get_manifest.sh 1.0.0
+
+# standard bash error handling
+set -o nounset  # treat unset variables as an error and exit immediately.
+set -o errexit  # exit immediately when a command fails.
+set -E          # needs to be set if we want the ERR trap
+set -o pipefail # prevents errors in a pipeline from being masked
+
+TAG=${1}
+MANIFEST_FILENAME="btp-manager.yaml"
+BTP_MANAGER_RELEASES_URL="https://github.com/kyma-project/btp-manager/releases"
+
+curl -sJL ${BTP_MANAGER_RELEASES_URL}/download/"${TAG}"/${MANIFEST_FILENAME}
```

Comment:
From Ubuntu's man for curl we lear that `-J` is not recommended: 
```WARNING:  Exercise  judicious  use  of  this option, especially on Windows. A rogue
              server could send you the name of a DLL or other file that could possibly be loaded
              automatically by Windows or some third party software.```

Reply:
1. We have full control on the url and filename here, so we exercise caution
2. We do not run it on Windows

Nevertheless I removed the `-J` switch.

Please create a ticket to remove `-J` switch from all curl invocations in our scripts/workflows.